html/public/templatePublic.twig

```diff
@@ -191,21 +191,21 @@
                         </li>
                         {% if INSTANCE.publicData.enableAssets %}
                             <li class="nav-item">
-                                <a class="nav-link" href="assets.php">
+                                <a class="nav-link" href="{{ INSTANCE.publicData.customDomains[0] }}/assets.php">
```

Comment:
I think currently you have `{{ INSTANCE.publicData.customDomains[0] }}/assets.php` but what if you get rid of the whole custom domain bit, and just have `/assets.php` is should fix the issue

Reply:
that way it does not matter what the domain is it is always correct

Reply:
Whilst walking back, I realised that I could just use a relatively link.
Ergo `../assets.php` should work surely. But then it isn't as flexible and I would want an absolute link.